source/quick-reference.txt

```diff
@@ -9,6 +9,17 @@ Quick Reference
 This page shows the driver syntax for several MongoDB commands and links to
 their related reference and API documentation.
 
+Compatibility
+-------------
+
+.. |page-topic| replace:: use the following driver syntax
```

Comment:
I: This page-topic makes the final sentence a little awkward, especially if you accept my suggestion in an earlier comment:

"You can use the Node.js driver to connect and [use the following driver syntax] for deployments hosted in the following environments:"

S:

```suggestion
.. |page-topic| replace:: execute commands
```

^^^ This is how the page intro describes the content of the page